container/inject.go

```diff
@@ -0,0 +1,19 @@
+package container
+
+import (
+	"fmt"
+
+	"github.com/golobby/container"
+	"github.com/oscarSantoyo/academy-go-q12021/service"
+)
```

Comment:
As a best practice, we should organize our imports in alphabetic order and taking first into consideration the following groups separated by an empty line:

First, Go's standard libraries (fmt)
Second, your local packages (github.com/oscarSantoyo/academy-go-q12021/service)
And lastly, 3rd party packages (github.com/golobby/container)

Something like this:
```suggestion
import (
	"fmt"

	"github.com/oscarSantoyo/academy-go-q12021/service"
	
	"github.com/golobby/container"
)
```